packages/frontend-2/components/settings/shared/DeleteUserDialog.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <LayoutDialog v-model:open="open" max-width="sm" :buttons="dialogButtons">
+    <template #header>Remove user</template>
+    <div class="flex flex-col gap-4 text-body-xs text-foreground">
+      <p>Are you sure you want to remove the following user from the workspace?</p>
+      <p class="font-semibold">
```

Comment:
font-medium

Reply:
Updated 👍 Also changed it in a couple other places in settings